po/cs.po

```diff
@@ -0,0 +1,182 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the lutris package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2012-11-09 21:49+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: Czech <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../data/ui/AboutDialog.ui.h:1
+msgid "About Lutris"
+msgstr "O Lutris"
+
+#: ../data/ui/AboutDialog.ui.h:2
+msgid "© 2010, 2012 Mathieu Comandon"
+msgstr "© 2010, 2012 Mathieu Comandon"
+
+#: ../data/ui/AboutDialog.ui.h:3
+msgid "Open Source Gaming Platform"
+msgstr "Herní platforma s otevřeným zdrojem"
```

Comment:
What about "Otevřená herní platforma"? "Otevřený zdroj" sounds strange to me (just my opinion).